core/arch/arm/plat-stm32mp2/main.c

```diff
@@ -207,8 +209,14 @@ void __noreturn plat_panic(void)
 
 static TEE_Result setup_multi_core_panic(void)
 {
-	itr_add(&sgi9_reset_handler);
-	itr_enable(sgi9_reset_handler.it);
+	TEE_Result res = TEE_ERROR_GENERIC;
+
+	res = interrupt_add_handler_with_chip(interrupt_get_main_chip(),
+					      &sgi9_reset_handler);
+	if (res)
+		panic();
+
+	interrupt_enable(sgi9_reset_handler.chip, sgi9_reset_handler.it);
```

Comment:
For commit "core: panic: platform specific panic sequence": 
These changes in _core/arch/arm/plat-stm32mp2/main.c_ should be squashed in commit "plat-stm32mp2: implement plat_panic() function" where they belong.

Reply:
Will remove this commit from this series